utils/src/rpc.rs

```diff
@@ -8,7 +8,6 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 static INFURA_KEYS: Lazy<Vec<&'static str>> = Lazy::new(|| {
     let mut keys = vec![
         "6770454bc6ea42c58aac12978531b93f",
```

Comment:
@mattsse just a heads up I think this key might also not work, I had to remove it in forge-std to get CI passing https://github.com/foundry-rs/forge-std/pull/284

Reply:
thanks, yeah this one broke as well, removing